### DISCUSSION TITLE
Welcome to Collect-your-GamingTools Discussions!

### DISCUSSION THREAD

**Discussion by swaraj-das** [swaraj-das/Collect-your-GamingTools#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DevROHIT11** [comment]
Hello !! 
@swaraj-das 

I really loved your website UI its pretty attractive !! I will love to contribute to your project.

**Reply by swaraj-das** [reply]
> Hello !! @swaraj-das
> 
> I really loved your website UI its pretty attractive !! I will love to contribute to your project.

Hey @DevROHIT11, thanks for the kind words! Feel free to raise any issues you find and make PRs to contribute. Looking forward to your contributions !

**Reply by DevROHIT11** [reply]
I just created the pull request for one of the issue of aligning return to top arrow and chatbot and My request is pending will you plz accept it

**Reply by swaraj-das** [reply]
I will check all the PRs. Don't worry. Provide your PR number.

**Reply by DevROHIT11** [reply]
#626

**Comment by DevROHIT11** [comment]
Heyy @swaraj-das  I had raised a Pull request #626 would you please check it when ever you get a chance . If their is need of any adjustment please ensure me I would happy to do it!! 
Thank you .

**Reply by swaraj-das** [reply]
> Heyy @swaraj-das I had raised a Pull request #626 would you please check it when ever you get a chance . If their is need of any adjustment please ensure me I would happy to do it!! Thank you .

Ok I will check.

**Reply by DevROHIT11** [reply]
Sorry #626 is my issue no.s and #615 is my Pr number

**Comment by Swastik0710** [comment]
Your website is wonderful.

**Comment by Swastik0710** [comment]
Keep up the good work. 😊
